src/app/contact/contact.component.html

```diff
@@ -5,11 +5,15 @@
 <nav class="navbar navbar-default nav-about">
   <div class="container-fluid">
     <div class="navbar-header">
-      <a class="navbar-brand about-navbar" href="//susper.com">
+      <a class="navbar-brand contact-navbar" href="//susper.com">
         <img alt="brand" class="navbar-logo" src="../../assets/images/susper.svg">
       </a>
     </div>
-    <p class="navbar-text navbar-right">Contact Us</p>
+    <p class="navbar-collapse navbar-right">
+      <a href="terms">Terms</a>
```

Comment:
this is a wrong way to navigate to a page. this will reload the whole page use `routerLink`